ht.access

```diff
@@ -18,6 +18,10 @@ RewriteBase /
 RewriteRule "/\.|^\.(?!well-known/)" - [F]
 
 
+# Prevent rewrite the .well-known directory used by LetsEncrypt by rules below of this rule
+RewriteRule "^\.well-known" - [L]
```

Comment:
```suggestion
RewriteRule "^\.well-known/" - [L]
```

Make sure that we match the directory `.well-known` and not the file and/or other directories starting with  `.well-known`. Although the previous rule already blocks this, it is still good to apply it here just in case.